File: broadcastreceiverlibrary/src/main/java/com/osapps/broadcastreceiverlibrary/sms/SmsObserver.java
package com.osapps.broadcastreceiverlibrary.sms;


import com.osapps.broadcastreceiverlibrary.generic.BroadcastObserver;

/**
 * Created by osapps on 07/05/2018.
 */
public interface SmsObserver extends BroadcastObserver {
    void onNewSms(NewSmsArray newSmsObj);
}

File: broadcastreceiverlibrary/src/main/java/com/osapps/broadcastreceiverlibrary/sms/SmsObserver.kt
package com.osapps.broadcastreceiverlibrary.sms


import com.osapps.broadcastreceiverlibrary.generic.BroadcastObserver

/**
 * Created by osapps on 07/05/2018.
 */
interface SmsObserver : BroadcastObserver {
    fun onNewSms(newSmsObj: NewSmsArray)
}
